hosts

```diff
@@ -25,6 +26,8 @@ wiley-ctlr-3 ansible_ssh_host=172.17.16.72
 [osp:children]
```

Comment:
Doesn't appear that the host group osp is used in a playbook anywhere.  Why are we adding host groups to this group?  And group swift does not exist.